src/main/webapp/instructorHelp.html

```diff
@@ -7461,7 +7568,7 @@ <h5 class="col-sm-2">
                                             Team 3
                                         </td>
                                         <td>
-                                            
+
```

Comment:
We don't have rules on empty lines. I prefer the change set to be minimum, so try and revert this kind of change.